components/responsive/src/Responsive.svelte

```diff
@@ -28,10 +35,26 @@
     });
   });
 
+  const getOrientation = (): OrientationType => {
+    if (window.screen.orientation) {
+      return screen.orientation.type;
+    }
+
+    // fallback to window.orientation (on iOS)
+    const orientations = new Map([
```

Comment:
can declare constant `orientations` out of the `getOrientation` function? bcs every time the function called, the constant would be declared, which is something that we might not want.